lib/actions/actions.js

```diff
@@ -35,12 +35,14 @@ actions._prepCopy = function _prepCopy(source, destination, process) {
   destination = this.isPathAbsolute(destination) ? destination : path.join(this.destinationRoot(), destination);
 
   var encoding = null;
-  var binary = isBinaryFile(source);
-  if (!binary) {
-    encoding = 'utf8';
-  }
+  var binary = true;
+
+  body = fs.readFileSync(source);
 
-  body = fs.readFileSync(source, encoding);
+  if (istextorbinary.isTextSync(undefined, body)) {
```

Comment:
Let's not change value of a variable inside the `if` statement. It's making it harder to reason about.

```js
isText = istextorbinary.isTextSync(undefined, body)

if (isText) {
  // foo
}

// etc
```